templates/README.template

```diff
@@ -1,112 +1,216 @@
 [![npm][npm]][npm-url]
+[![node][node]][node-url]
 [![deps][deps]][deps-url]
-[![test][test]][test-url]
+[![tests][tests]][tests-url]
 [![coverage][cover]][cover-url]
 [![chat][chat]][chat-url]
 
 <div align="center">
   <!-- replace with accurate logo e.g from https://worldvectorlogo.com/ -->
-  <img width="200" height="200" src="https://cdn.worldvectorlogo.com/logos/javascript.svg">
+  <img width="140" height="140" src="https://cdn.worldvectorlogo.com/logos/javascript.svg">
```

Comment:
Why project icon is smaller than webpack icon? ;-) I’d make `hspace` the same, otherwise they won’t be centered correctly.

Reply:
@michael-ciniawsky - Just use this one

```
  <img width="185" height="185" src="https://cdn.worldvectorlogo.com/logos/nodejs-icon.svg">
  <a href="https://webpack.js.org/">
    <img width="200" height="200" src="https://webpack.js.org/assets/icon-square-big.svg">
  </a>

```

Reply:
Done :)